runtime/pom.xml

```diff
@@ -10,8 +10,8 @@
 	</parent>
```

Comment:
Don't we need to update java classes that refer to javax to use jakarta?
Eg:
import javax.xml.bind.ValidationEventLocator; in ObjectLocator.java?

Reply:
Hi @cdwijayarathna : the goal here is to prepare jakarta migration of this plugin.
@mattrpav is on the job of aligning the 3 majors projects of @highsource in order to release a final javax version (2.X) before going to jakarta (3.x for java 8+ - 4.x for java 11+)